docs/guides/latest-updates.mdx

```diff
@@ -1,271 +1,278 @@
 
-#### New features
+- New features
 
-qiskit-ibm-runtime is now compatible with Qiskit v2.0. This means that classes and methods removed in Qiskit v2.0 have also been removed or handled accordingly.  The `pulse` and `channel` related changes are highlighted in the following list. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.
+  - `qiskit-ibm-runtime` is now compatible with Qiskit v2.0. This means that classes and methods removed in Qiskit v2.0 have also been removed or handled accordingly.  The `pulse`- and `channel`-related changes are highlighted in the following list. See the [Pulse migration guide](/migration-guides/pulse-migration) for details.
 
-   - `Channel` methods in `IBMBackend` have been removed.
-   - The backend configuration class `PulseBackendConfiguration` has been removed, so all backends will now be returned as `QasmBackendConfiguration`.
-   - `PulseDefaults` (backend defaults) can still be retrieved but they are no longer necessary when creating a backend `Target`.
+    - `Channel` methods in `IBMBackend` have been removed.
+    - The backend configuration class `PulseBackendConfiguration` has been removed, so all backends will now be returned as `QasmBackendConfiguration`.
+    - `PulseDefaults` (backend defaults) can still be retrieved but they are no longer necessary when creating a backend `Target`.
 
+- See the full [0.37.0 release notes](/api/qiskit-ibm-runtime/release-notes#0370-2025-03-13).
 
-See the full [0.37.0 release notes](/api/qiskit-ibm-runtime/release-notes#0370-2025-03-13).
 {/* use url format /api/qiskit-ibm-runtime/release-notes#0XYZ to link to the specific 0.XY.Z version of the release notes */}
 
-### Qiskit Transpiler Service client v0.3.0
+- *To see the release notes for all versions, visit the [Qiskit Runtime client release notes](/api/qiskit-ibm-runtime/release-notes).*
 
-We’re pleased to share that the beta release of the Qiskit Transpiler Service is now available to all IBM Quantum&reg; Premium Plan users.
+<span id="latest-runtime-service"></span>
+## Qiskit Runtime service
 
-The Qiskit Transpiler Service leverages the resources of IBM Cloud&reg; to provide users with the latest transpilation capabilities from the Qiskit SDK. It offers a Python library that helps users seamlessly integrate the service into their current Qiskit patterns and workflows. Perhaps most importantly, the service invites users to experiment with new and improved AI-powered transpiler passes — cutting-edge tools that might be faster and produce better results than traditional transpilation methods.
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
```

Comment:
```suggestion
This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (by using `qiskit-ibm-runtime` or otherwise), or which version of the client SDK you use.
```